updating map castle of trouble

farooqkz opened the pull request:
I've updated "Castle of Trouble" in the hope to have it back.

 - There are prebuilt bases of obsidian for both teams.
 - The journey between bases must be longer and harder now

LuismiInTheZone commented:
I can't walk out of the base, it says that I can pass the barrier before starting... Also... what did you changed? Because it seems like it's almost the same... You added two walls and bases and... something more?

farooqkz commented:
> I can't walk out of the base, it says that I can pass the barrier before starting... Also... what did you changed? Because it seems like it's almost the same... You added two walls and bases and... something more?

I will look into the issue you've mentioned later. For now those two changes are the only changes and I hope that they are enough to make capturing hard enough :)

farooqkz commented:
@MoreKat I suggest you remove the "New map" label as its enhancement for an old map.

farooqkz commented:
I need help solving the issue @LuismiInTheZone mentioned. I've set the team zones but still having the same problem.

farooqkz commented:
@HobbitPower help needed

farooqkz commented:
Problems solved. Ready for review

LuismiInTheZone commented:
Exactly the same problem like the other map, Plain 3, I can't stand on some of the nearest blocks next to the barrier, also, what did you changed this time?

farooqkz commented:
> Exactly the same problem like the other map, Plain 3, I can't stand on some of the nearest blocks next to the barrier, also, what did you changed this time?

Solved

LuismiInTheZone commented:
I can cross some barriers a bit before starting the mach, check again the map and make sure to put the exact location of each thing at the barrier.

farooqkz commented:
> I can cross some barriers a bit before starting the mach, check again the map and make sure to put the exact location of each thing at the barrier.

I've made the team zones overlap a bit. Isn't it fine?

farooqkz commented:
@HobbitPower Ready for review

LuismiInTheZone commented:
> @HobbitPower Ready for review

I don't see any changes since last update here at github

farooqkz commented:
> > @HobbitPower Ready for review
> 
> I don't see any changes since last update here at github

Should there be?

LuismiInTheZone commented:
> > > @HobbitPower Ready for review
> > 
> > 
> > I don't see any changes since last update here at github
> 
> Should there be?

If there's an error... We need the files to be correct without errors

farooqkz commented:
> > > > @HobbitPower Ready for review
> > > 
> > > 
> > > I don't see any changes since last update here at github
> > 
> > 
> > Should there be?
> 
> If there's an error... We need the files to be correct without errors, so you need to upload some new version without them

There is no error and everything fine from my point of view.

LuismiInTheZone commented:
> I can cross some barriers a bit before starting the mach, check again the map and make sure to put the exact location of each thing at the barrier.

@farooqkz I tested the map, and this is wrong

LuismiInTheZone commented:
> Oh that. Is that really a problem?

It's the red barrier, so yes

farooqkz commented:
The problem mentioned by Luis should've been solved now. Waiting for review from Sir @HobbitPower :)

LuismiInTheZone commented:
I don't know what to say... I think the map keeps being very short...

farooqkz commented:
Bases made of obsidian will really reduce attempts.

MoreKat commented:
I suggest adding a better screenshot, one without the barrier visible and one that gives a better overview of the map.

farooqkz commented:
> I suggest adding a better screenshot, one without the barrier visible and one that gives a better overview of the map.

Done

HobbitPower commented:
> > I suggest adding a better screenshot, one without the barrier visible and one that gives a better overview of the map.
> 
> Done

A screenshot in 3:2 ratio

HobbitPower commented:
> I can't walk out of the base, it says that I can pass the barrier before starting... Also... what did you changed? Because it seems like it's almost the same... You added two walls and bases and... something more?

@LuismiInTheZone was that fixed the last time you checked?
I still get an error there: 
![image](https://user-images.githubusercontent.com/76528136/235881121-b2147416-f989-4aa7-b5a5-d3229e072a33.png)

farooqkz commented:
> > I can't walk out of the base, it says that I can pass the barrier before starting... Also... what did you changed? Because it seems like it's almost the same... You added two walls and bases and... something more?
> 
> @LuismiInTheZone was that fixed the last time you checked? I still get an error there: ![image](https://user-images.githubusercontent.com/76528136/235881121-b2147416-f989-4aa7-b5a5-d3229e072a33.png)

strange I don't get this error on my test server(`test.minetest.ir:30000`)

farooqkz commented:
> > > I suggest adding a better screenshot, one without the barrier visible and one that gives a better overview of the map.
> > 
> > 
> > Done
> 
> A screenshot in 3:2 ratio

hmm my original screenshot is not 3:2 :/ attached
![screenshot](https://user-images.githubusercontent.com/15038218/235889370-1b1c693b-f35e-4516-ad80-dff10f69fea9.png)

a-blob commented:
I want to say something about the addition of the obsidian base: the obsidian basses might prevent the matches from ending too fast, but the appearance of the obsidian basses looks a bit odd, especially within the setting of the castle. No offense or anything, but the obsidian base looks a bit unnatural and ugly, and it doesn't seem to fit. Maybe it would look better if details are added. Basses don't have to look good, but now its part of the map, which is different.
The current build reminds me a bit of these:
![image](https://user-images.githubusercontent.com/88883098/236089434-70243dc2-3f26-4de8-8851-5860f744264f.png)

farooqkz commented:
> I want to say something about the addition of the obsidian base: the obsidian basses might prevent the matches from ending too fast, but the appearance of the obsidian basses looks a bit odd, especially within the setting of the castle. No offense or anything, but the obsidian base looks a bit unnatural and ugly, and it doesn't seem to fit. Maybe it would look better if details are added. Basses don't have to look good, but now its part of the map, which is different. The current build reminds me a bit of these: ![image](https://user-images.githubusercontent.com/88883098/236089434-70243dc2-3f26-4de8-8851-5860f744264f.png)

Never thought it this way and yes, you are right. What do you recommend?

a-blob commented:
> Never thought it this way and yes, you are right. What do you recommend?

I would try to mimic the style of the castle and rebuild the base so that it fits in a bit more. Adding details and making it so that the base is not a perfect rectangular prism might help.

farooqkz commented:
> > Never thought it this way and yes, you are right. What do you recommend?
> 
> I would try to mimic the style of the castle and rebuild the base so that it fits in a bit more. Adding details and making it so that the base is not a perfect rectangular prism might help.

That's a good idea but the problem's that obsidian is almost unbreakable(to be precise: very hard to break). But castle blocks are not even half that strong. Do you think it's a good idea to use unbreakable blocks for the bases?

a-blob commented:
I think you can still end up with a pretty decent base using only Obsidian related blocks - since the game offers Obsidian stairs and slabs and other things. I don't think it is a good idea to use large amounts of unbreakable blocks for the bases.

farooqkz commented:
> I think you can still end up with a pretty decent base using only Obsidian related blocks - since the game offers Obsidian stairs and slabs and other things. I don't think it is a good idea to use large amounts of unbreakable blocks for the bases.

I thought you meant an obsidian base's look is paradoxical with the rest of the castle.

a-blob commented:
> I thought you meant an obsidian base's look is paradoxical with the rest of the castle.

Of course using castle blocks would make it look even better. But  Obsidian seems like a better choice for this scenario since it is much stronger.

farooqkz commented:
> > I thought you meant an obsidian base's look is paradoxical with the rest of the castle.
> 
> Of course using castle blocks would make it look even better. But Obsidian seems like a better choice for this scenario since it is much stronger.

What about unbreakable castle blocks then?

LuismiInTheZone commented:
> > I can't walk out of the base, it says that I can pass the barrier before starting... Also... what did you changed? Because it seems like it's almost the same... You added two walls and bases and... something more?
> 
> @LuismiInTheZone was that fixed the last time you checked? I still get an error there: ![image](https://user-images.githubusercontent.com/76528136/235881121-b2147416-f989-4aa7-b5a5-d3229e072a33.png)

Huh, I remember that fixed, I don't know...

LuismiInTheZone commented:
> > I want to say something about the addition of the obsidian base: the obsidian basses might prevent the matches from ending too fast, but the appearance of the obsidian basses looks a bit odd, especially within the setting of the castle. No offense or anything, but the obsidian base looks a bit unnatural and ugly, and it doesn't seem to fit. Maybe it would look better if details are added. Basses don't have to look good, but now its part of the map, which is different. The current build reminds me a bit of these: ![image](https://user-images.githubusercontent.com/88883098/236089434-70243dc2-3f26-4de8-8851-5860f744264f.png)
> 
> Never thought it this way and yes, you are right. What do you recommend?

I don't know about your opinion... I still feel that's going to change nothing

farooqkz commented:
> > > I want to say something about the addition of the obsidian base: the obsidian basses might prevent the matches from ending too fast, but the appearance of the obsidian basses looks a bit odd, especially within the setting of the castle. No offense or anything, but the obsidian base looks a bit unnatural and ugly, and it doesn't seem to fit. Maybe it would look better if details are added. Basses don't have to look good, but now its part of the map, which is different. The current build reminds me a bit of these: ![image](https://user-images.githubusercontent.com/88883098/236089434-70243dc2-3f26-4de8-8851-5860f744264f.png)
> > 
> > 
> > Never thought it this way and yes, you are right. What do you recommend?
> 
> I don't know about your opinion... I still feel that's going to change nothing, too short

I don't have much of an opinion but I think it's worth giving a try.

a-blob commented:
> What about unbreakable castle blocks then?

Hmm, not sure; that would be your decision.

farooqkz commented:
Guys, I've made an update :)

olliy1or commented:
team zones doesn't seem to be correct, and there are no team chests

farooqkz commented:
> team zones doesn't seem to be correct, and there are no team chests

fixed

farooqkz commented:
@olliy1or @HobbitPower  any other change needed?

LuismiInTheZone commented:
Gonna say it again: Map keeps been very small

farooqkz commented:
> Gonna say it again: Map keeps been very small, I don't think bases is gonna change much

I think we better try it at least

HobbitPower commented:
> @olliy1or @HobbitPower any other change needed?

For me, the main problem is still the size of the map and indestructible bases are not an ideal solution in my opinion. Could lead to base camping and spawn-killing at the same time. So I don't really see the PR as an improvement to the map. However, I'm open to consult about this with my fellow map managers.

For now: you should add ores if you want to enable it for classic. I found 4 mese while testing, so you could add some iron ores for example.

farooqkz commented:
> > @olliy1or @HobbitPower any other change needed?
> 
> For me, the main problem is still the size of the map and indestructible bases are not an ideal solution in my opinion. Could lead to base camping and spawn-killing at the same time. So I don't really see the PR as an improvement to the map. However, I'm open to consult about this with my fellow map managers.
> 
> For now: you should add ores if you want to enable it for classic. I found 4 mese while testing, so you could add some iron ores for example.

I am open to suggestions to improve the map

HobbitPower commented:
> I am open to suggestions to improve the map

 You could enlarge the map for example. Or maybe you can think of something else, but so far I don't see a good reason for a merge.

farooqkz commented:
> > I am open to suggestions to improve the map
> 
> You could enlarge the map for example. Or maybe you can think of something else, but so far I don't see a good reason for a merge.

Can you ask more map makers on Discord on how to improve the map? I am also thinking to making a maze with indestructible or hardly destructible blocks so that the journey between bases will take longer.

src4026 commented:
I had a look at the map, and would like to cover various aspects of it below:
- Architecture: Nice work
- Playability: The map is small and can easily be captured with enough outmanoeuvring players. If you are planning to make a maze or some other way of increasing the difficulty, you will still probably have to enlarge it. I am open to seeing other ideas too. I will also try to think of some in the meanwhile.
- Blocks: I am not comfortable knowing that _everything_ is indestructible, including the base, it seems way too safe and dangerous, including the leaves of the trees and the tree itself. There could be certain cases of deaths by the ones in the map somehow.
Not my final thoughts on this, I shall come back to this later :+1:

farooqkz commented:
I thank everyone for their feedback, in here and on Discord.

olliy1or commented:
> I thank everyone for their feedback, in here and on Discord.

i assume you'll be working on this map further, so i'll adjust the lables appropriately. let me know if i got this wrong

HobbitPower commented:
@farooqkz https://github.com/MT-CTF/maps/pull/307#issuecomment-2198067229 I think that also applies to the map. I would close the PR for now.

farooqkz commented:
> @farooqkz [#307 (comment)](https://github.com/MT-CTF/maps/pull/307#issuecomment-2198067229) I think that also applies to the map as well? If so, I would close the PR for the time being, because of the inactivity (which is likely to continue).

Agreed.

HobbitPower commented:
I close this due to inactivity and no prospect of change in the near future. If there is interest in working on the PR again, it can be re-opened.
